### DISCUSSION TITLE
ability to increase grid spacing further from 128 to 512 (or option to disable grid)

### DISCUSSION THREAD

**Discussion by rikiaosu** [ppy/osu#29713] [Ideas] [answered]
hello, i searched for "grid", "editor grid" and "grid size" and couldn't find any discussion that contained what my request is
![image](https://github.com/user-attachments/assets/4d655c2b-2708-419e-bf6d-a8fc27e09412)
currently in the editor the biggest grid size you can get is 12 separate sections, but as someone who doesn't ever use the grid it would be nice to be able to disable the grid for visual clarity

[someone on twitter](https://x.com/wHydria/status/1831795246579249570/) seemed to be able to accomplish this,
![GWvX9TeWoAAXE13](https://github.com/user-attachments/assets/cb995105-aa0b-4d1d-ad7d-633ac9aa530a)
![GWvX_5NWUAA8b3K](https://github.com/user-attachments/assets/20b4deb6-9caa-4128-bc9a-e848b91ad658)

 so i hope this isn't too big of an ask despite being probably a very useless request overall (idk if you want to add a checkbox that essentially sets it to a huge value that it disappears and then you can control x and y offset to make the centre axis disappear as well) i think just increasing the limit for grid spacing from 128 to 512 would make it disappear completely (or add a option for me to like double click the number and manually input a value outside what the slider provides would be the same) i also did try to "Edit externally" the map and change grid size to 512 that way but it also did not work unfortunately
 
![image](https://github.com/user-attachments/assets/f774fcc1-8e90-4d00-9d63-bbd4ec798e18)

i sincerely apologize if this kind of counts as spam but after mapping for nearly 8 years this has bothered me for so long and i would love to see the option to turn it off in lazer, thank you so much

**Comment by bdach** [comment]
I'm not sure I can get behind fixing this by increasing grid spacing. A toggle to hide grid (in the view menu up top) sounds preferable to me.

**Reply by rikiaosu** [reply]
sure, either method is fine by me. i just wanted to offer multiple options to achieve this for whichever way works. even if just adding a secret bypass by editing the .osu to achieve this i would be fine with and it would affect nobody else

**Reply by rikiaosu** [reply]
hey it's been like half a year or so i've been wondering if y'all have been able to add that hide grid toggle yet?

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/32311
